spot_hardware_interface/include/spot_hardware_interface/spot_hardware_interface.hpp

```diff
@@ -86,10 +98,12 @@ class StateStreamingHandler {
    */
   void handle_state_streaming(::bosdyn::api::RobotStateStreamResponse& robot_state);
   /**
-   * @brief Get a struct of the current joint states of the robot.
+   * @brief Get structs of the current joint states and IMU data from the robot.
    * @return JointStates struct containing vectors of position, velocity, and load values.
+   * ImuStates struct containing info on the IMU's identifier, mounting link, position, linear acceleration,
+   * angular velocity, and rotation
    */
-  void get_joint_states(JointStates& joint_states);
+  void get_states(JointStates& joint_states, ImuStates& imu_states);
```

Comment:
A quick question, is there any code that used the `get_joint_states` method that has to be also updated with this change? Is this a breaking change?